sympy/sets/__init__.py

```diff
@@ -3,3 +3,7 @@
 from .fancysets import ImageSet, Range, ComplexRegion
 from .contains import Contains
 from .conditionset import ConditionSet
+
+from ..core.singleton import S
```

Comment:
You might try asking on StackOverflow about this. A quick Google search for the error message didn't turn anything up.